code/game/machinery/doors/brigdoors.dm

```diff
@@ -118,34 +114,37 @@
 			return
 	atom_say("[src] beeps, \"[occupant]: [notifytext]\"")
 
-/obj/machinery/door_timer/Initialize()
-	..()
+/obj/machinery/door_timer/Initialize(mapload)
+	. = ..()
 
+	GLOB.celltimers_list += src
 	Radio = new /obj/item/radio(src)
 	Radio.listening = 0
 	Radio.config(list("Security" = 0))
 	Radio.follow_target = src
+	return INITIALIZE_HINT_LATELOAD
 
-	spawn(20)
-		for(var/obj/machinery/door/window/brigdoor/M in GLOB.airlocks)
-			if(M.id == id)
-				targets += M
+/obj/machinery/door_timer/LateInitialize()
+	. = ..()
+	for(var/obj/machinery/door/window/brigdoor/M in GLOB.airlocks)
```

Comment:
`GLOB.airlocks` is currently populated in `/obj/machinery/door/New()`, so *technically* this code can go into `Initialize()`.
However ideally we'd axe the `door/New()` eventually and have to use LateInitialze here anyway, so might as well do this now.